web/samples_index/lib/src/samples.yaml

```
@@ -633,3 +633,23 @@ samples:
     tags: ['demo', 'animation']
     web: https://z.flutter.gallery/#/dice
     type: demo
+
+  - name: Web Embedding
+    author: Flutter
```

```suggestion
    author: Flutter and Angular
```

Mark Thompson from Angular helped build this, let's give them credit too!

---

Good catch!